README.md

```diff
@@ -13,7 +13,12 @@ workflow. For more information on getting started, see
 
 The production instance is running in
 [CentOS CI](https://jenkins-fedora-coreos.apps.ci.centos.org)
-(though note anonymous view is blocked by default).
+(though note anonymous view is blocked by default). Its raw
+build output can be seen in the
+[build browser](https://builds.coreos.fedoraproject.org/browser)
+(but note that the latest *supported* version of FCOS must
```

Comment:
might make sense to bold `supported` as well

Reply:
Ahh, I mixed up italics and bold. Well, now it's both!